Exonoesis/Helping-Hand
# Update Bevy and dependencies

## What is the purpose of these changes?
It's important for the project to be kept up to date and using the latest versions of Bevy and all third-party libraries.

## What has changed?
Updated Bevy to 0.13, LDtk to the latest commit plus a patch, Kira Audio to 0.19, the LDtk app version to 1.5.3, and added cucumber to the project for more streamlined workflow.

## How can I verify this works?
1. Open the project in editor of your choice.
2. Run tests and verify they pass.
3. Launch application and see that it works as it did before the update, able to enter the settings menu, change the audio sliders, enter the play state, move around as before, collision is working as intended, music and SFX play as intended, and the quit button is still functional.

This PR closes issue #52
